Apps/WebClient/src/ClientApp/src/components/DependentCardComponent.vue

```diff
@@ -11,7 +11,7 @@ import Vue from "vue";
 import { Component, Emit, Prop, Ref } from "vue-property-decorator";
 import { Action, Getter } from "vuex-class";
 
-import Covid19LaboratoryTestDescriptionComponent from "@/components/laboratory/covid19LaboratoryTestDescription.vue";
+import Covid19LaboratoryTestDescriptionComponent from "@/components/laboratory/covid19LaboratoryTestDescriptionComponent.vue";
```

Comment:
Is there a reason why this camel case vs pascal case?  Looks like most of the files had been renamed using pascal case.